DEVELOPMENT.md

```diff
@@ -493,45 +493,107 @@ build times. If you need to update this image, proceed as follows:
    create a new branch and build the updated image.
 
 ## Releases
+
+This section describes how to release a new version of Upstream.
+
 ### Prerequisites
-#### Google Cloud CLI
 
-For uploading artifacts to releases.radicle.xyz, you'll need a working `gcloud`
-environment. To do so, follow points 1 and 2 from the
-[Docker image updates][do] section.
+* [`gcloud`][gc] to upload artifacts. You need to ask for access to the
+  `releases.radicle.xyz` storage bucket. See also points 1 and 2 from the
+  [Docker image updates][do] section.
+* [`hub`][hb] version >= 2.14 to interact with GitHub’s API. See [its
+  documentation][hub-config] on how to configure access
+* [`brew`][br] to update the Uptream cask.
+
+All Github access tokens _must_ have the `public_repo` scope.
+
+### Process
+
+1. Create release candidate
+    1. Create a release candidate branch with  a commit that updates the version
+       and changelog and create a pull-request
+
+       ```bash
+       ./scripts/release.ts create-rc patch
+       ```
+
+    2. Open a draft pull request on [`radicle.xyz`](http://radicle.xyz) to
+       update latest version.
 
+       ```bash
+       cd radicle.xyz
+       git fetch --all
+       git checkout -b update-latest-release origin/master
+       echo -n X.Y.Z > partials/upstream-version.mustache && make
+       git commit --all --message "Point to latest upstream release"
+       hub pull-request --push --draft --no-edit
+       ```
 
-#### GitHub `hub` CLI tool
+2. Test the release
+    1. Wait for CI of the release candidate PR to pass.
+    2. Publish the CI artifacts as release candidate binaries.
 
-Please install the [`hub`][hb] CLI tool (version >= **2.14**), we use it in our
-release automation script to:
+       ```bash
+       ./sripts/release.ts publish-rc-binaries
+       ```
 
-  - create a pull-request off of a release branch;
-  - to merge the release branch into master;
-  - to close the pull-request.
+    3. Create QA issues for the release that link to the release candidate
+       binaries.
 
-Then you'll have to create a _Personal access token_ for it in the
-[GitHub Developer settings][gt] page and authenticate the CLI tool once
-by running any command that does a request to GitHub, like so: `hub api`.
-You'll be asked to provide your GitHub login and the access token.
+       ```bash
+       ./sripts/release.ts create-qa-issues
+       ```
 
-#### Homebrew CLI
+    4. Test the release by walking through the QA issues.
+    5. (Optional) To fix bugs, create a PR with the based on the release
+       candidate branch. Once it has been approved, fast-forward merge it into
```

Comment:
Instead of adding an example I’ve linked to the description in [`radicle-decisions`](https://github.com/radicle-dev/radicle-decisions/blob/master/proposals/0003.md#merging-pull-requests).